scripts/irods/test/test_ibun.py

```diff
@@ -286,3 +286,59 @@ def count_objects_in_collection(collection_path):
             lib.remove_resource(self.admin, compound_resource)
             lib.remove_resource(self.admin, cache_resource)
             lib.remove_resource(self.admin, archive_resource)
+
+    @unittest.skip('Generation of large file causes I/O thrashing... skip for now')
+    def test_ibun__issue_7474(self):
```

Comment:
Needs more words?

Reply:
I just copied what Alan had on his tests.

Reply:
My bad. I'm referring to the test name.

Reply:
Added more to the test name.